content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -186,23 +208,23 @@ function doWork(parent) {
 
 ## Span events
 
-An event is a human-readable message attached to a span that represents "something happening" during its lifetyime. You can think of it like a primitive log.
+An event is a human-readable message attached to a span that represents "something happening" during its lifetime. You can think of it like a primitive log.
 
 ```js
-span.addEvent('Doing something');
+span.addEvent("Doing something");
 
-const result = doWork()
+const result = doWork();
 
-span.addEvent('Did something');
+span.addEvent("Did something");
 ```
 
 You can also add an object with more data to go along with the message:
 
 ```js
-span.addEvent('some log', {
-  'log.severity': 'error',
-  'log.message': 'Data not found',
-  'request.id': requestId,
+span.addEvent("some log", {
```

Comment:
In otel-js repo, we use the single quote marks for string literals.

Reply:
Fixed